### PR TITLE
added amplitde_peak and working mysql table insertion gist

### PR DESCRIPTION
**Pull request by mattlianje**
to test:
- Have your Apache Web Server and MySql server running locally (MAMP, XAMP ...)
- `$ python main.py log`
- Output:
`amplitude_peak` is a binary variable ... 1 when the timestamp is at an amplitude peak, 0 o/w.
image

### CODE REVIEW

File: audio-analysis/main.py [hunk 1]
```diff
@@ -37,6 +39,11 @@ def analyzeVideoSound(url, tag):
         else:
             print('Join failed ... each word does not have an amplitude')
         print(df3)
+        df3.to_csv("data_export.csv", header=True)
+
+        engine = create_engine('mysql+pymysql://root:root@localhost/climactic_test')
+        df3.to_sql("test_table", con=engine, if_exists='replace')
+
```
Comment (ewega) [inline audio-analysis/main.py:46]: So for anyone looking at this in the future - the first "root" in _root:root@localhost/climactic_test_ is the username and the second is the password. You might have to remove the root after the colon if your localhost/server does not require a password.

**Review by ewega — APPROVED** [review]
Lookin nifty skrt

**Comment by tjlam** [comment]
Might be a dumb question but. Everytime we run the script does it wipe the old db data, generate the new data and put that into a new db? or does the old data persist? Will this be able to handle duplicate data?

**Comment by mattlianje** [comment]
@tjlam yep ... This is just hello world that overwrites. @Eldrick19 is writing the dedupe logic that will append to table.

File: audio-analysis/main.py (hunk 1 above)
Comment (tjlam) [reply audio-analysis/main.py]: > @tjlam yep ... This is just hello world that overwrites. @Eldrick19 is writing the dedupe logic that will append to table.

Nice, would it make sense to create a unique key to help with deduping? maybe a combo of youtube url and word number?

**Comment by tjlam** [comment]
> @tjlam yep ... This is just hello world that overwrites. @Eldrick19 is writing the dedupe logic that will append to table.

Nice, would it make sense to create a unique key to help with deduping? maybe a combo of youtube url and word number?

**Comment by ewega** [comment]
We were thinking of adding an auto incremented unique key in the table, and using a basic query every time a new YouTube video wants to be imported:

**IF** YouTube URL appears in table at least once **THEN** don't import YouTube URL in table.

**Comment by Eldrick19** [comment]
> > @tjlam yep ... This is just hello world that overwrites. @Eldrick19 is writing the dedupe logic that will append to table.
> 
> Nice, would it make sense to create a unique key to help with deduping? maybe a combo of youtube url and word number?

**Comment by tjlam** [comment]
> We were thinking of adding an auto incremented unique key in the table, and using a basic query every time a new YouTube video wants to be imported:
> 
> **IF** YouTube URL appears in table at least once **THEN** don't import YouTube URL in table.

Sounds good. Depending on how many youtube vids this might get slow b/c we're looking though every row for a youtube url. Then it would be better to index on youtube url and auto increment. Should be good for now though.

**Review by tjlam — APPROVED** [review]

**Pull request merged**